### PR TITLE
Fixes people not being able to respond to IRC admin PMs

### PR DESCRIPTION
**Pull request by DominikPanic**
Works
https://puu.sh/Dup2m/d05f29de41.png

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1016,6 +1016,24 @@
 
 	else if(istype(whom, /client))
 		recipient = whom
+		
+
+
+	if(irc)
```
Comment (spookydonut) [inline code/modules/admin/admin_verbs.dm]: is this discord or irc

Comment (DominikPanic) [reply code/modules/admin/admin_verbs.dm]: The convention on /tg/ is to call anything external related IRC.

Comment (spookydonut) [reply code/modules/admin/admin_verbs.dm]: do we have working irc code?

Comment (DominikPanic) [reply code/modules/admin/admin_verbs.dm]: Yes. This both works with Discord  and irc. It only sends a message to TGS which then decides which channel to use (either irc or discord.)

**Pull request merged**